sdk/lib/atl/atlimage.h

```diff
@@ -1057,6 +1059,8 @@ class CImage
     // Deprecated. Don't use this
     static const GUID *FileTypeFromExtension(LPCTSTR dotext)
     {
+        CImage dummy; // Initialize common
+
```

Comment:
```suggestion
        CImage dummy; // Initialize common
```
If this "dummy" thing is the actual hack, maybe you should prefix `HACK: ` in from of these comments.

Reply:
`"HACK:"` is added.